CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 ## Piwik 2.12.0
 
 ### Breaking Changes
+* Visitor details now additionally contain: `deviceTypeIcon`, `deviceBrand` and `deviceModel`
```

Comment:
It's more an improvement. But I thought it would be worth mentioning in changelog and we didn't have another headline it would fit in.